twint/src/main/java/com/adyen/checkout/twint/TwintComponent.kt

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2024 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 10/7/2024.
+ */
+
+package com.adyen.checkout.twint
+
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.adyen.checkout.action.core.internal.ActionHandlingComponent
+import com.adyen.checkout.action.core.internal.DefaultActionHandlingComponent
+import com.adyen.checkout.action.core.internal.ui.GenericActionDelegate
+import com.adyen.checkout.components.core.PaymentMethodTypes
+import com.adyen.checkout.components.core.internal.ButtonComponent
+import com.adyen.checkout.components.core.internal.ComponentEventHandler
+import com.adyen.checkout.components.core.internal.PaymentComponent
+import com.adyen.checkout.components.core.internal.PaymentComponentEvent
+import com.adyen.checkout.components.core.internal.toActionCallback
+import com.adyen.checkout.components.core.internal.ui.ComponentDelegate
+import com.adyen.checkout.core.AdyenLogLevel
+import com.adyen.checkout.core.internal.util.adyenLog
+import com.adyen.checkout.twint.internal.provider.TwintComponentProvider
+import com.adyen.checkout.twint.internal.ui.DefaultTwintDelegate
+import com.adyen.checkout.twint.internal.ui.TwintDelegate
+import com.adyen.checkout.ui.core.internal.ui.ButtonDelegate
+import com.adyen.checkout.ui.core.internal.ui.ComponentViewType
+import com.adyen.checkout.ui.core.internal.ui.ViewableComponent
+import com.adyen.checkout.ui.core.internal.util.mergeViewFlows
+import kotlinx.coroutines.flow.Flow
+
+/**
+ * A [PaymentComponent] that supports the [PaymentMethodTypes.TWINT] payment method.
+ */
+class TwintComponent internal constructor(
```

Comment:
This one should also not allow skipping the list of payment methods right? We need to rethink about the implementation of that logic

Reply:
Twint is already included in this logic, because it's in `SUPPORTED_ACTION_ONLY_PAYMENT_METHODS`